### PR TITLE
Edit logs article

### PR DESCRIPTION
**Pull request by storyaddict**
This PR does the following:

- Removes suggestion for emailing logs instead of attaching them to a ticket. Per @thomas-zimmerman 's request.
- Adds a section and screenshot for the new Support section in settings.
- Suggests using Google Drive or Wormhole in the case of abnormally large log files.

### CODE REVIEW

File: content/log-files.md [hunk 1]
```diff
@@ -19,26 +19,38 @@ tableOfContents: true
...
 
-Next click on the button outlined in red in the <u>System76 Driver</u> application and there will be a file called `system76-logs.tgz` placed in your Home directory (/home/username)
+Next click on the button outlined in red in the <u>System76 Driver</u> application and a file called `system76-logs.tgz` will be created and placed in your Home directory (`/home/username`).
 
 ![CreateLogFiles](/images/system76-driver/CreateLogFiles.png)
 
 ### Manually Generating Log Files
 
-If for some reason you are unable to access the <u>System76 Driver</u> application, you can manually create a set of logs files by running a series of commands via the Terminal application. To do this, click <u>Activities</u> in the top left of your system, search for <u>Terminal</u>, then open the application that appears.
+If for some reason you are unable to access the <u>System76 Driver</u> application, you can manually create a set of logs files by running a series of commands via the <u>Terminal</u> application. To do this, click `Activities`/`Applications` in the top left of your system, search for "Terminal", then open the application that appears.
```
Comment (jacobgkau) [inline content/log-files.md:47]: Should it be "log files" instead of "logs files?"

**Review by jacobgkau — COMMENTED** [review]
Good changes overall, I've left some notes.

File: content/log-files.md [hunk 2]
```diff
@@ -19,26 +19,38 @@ tableOfContents: true
...
 * A listing of the filesystems connected to your system, and the space used
 * Information about your firmware
 * A list of disks connected to your system, as well as connected USB devices
-* Sensors information showing temperatures and fanspeeds
+* Sensors information showing temperatures and fan-speeds
 * Comprehensive output from `syslog` and `journalctl`
 
+## Generating Log Files in Settings (Pop!_OS 21.10+)
+
+Starting in Pop!\_OS 21.10, logs are no longer generated through the <u>System76 Driver</u> application. Instead, a "Support" section has been added to the <u>Settings</u> application. Clicking the `Create Log Files` button will prompt you for your admin password, and then generate and save a file called `system76-logs.tgz`. This file is saved in your Home directory (`/home/username`).
```
[the remark this answers is not in the document]
Comment (storyaddict) [reply content/log-files.md:33]: Ah, thank you. I'll fix that!

File: content/log-files.md (hunk 1 above)
Comment (storyaddict) [reply content/log-files.md:47]: Ha, yes. It should be either "logs" or "log files."

File: content/log-files.md [hunk 3]
```diff
@@ -65,4 +77,6 @@ As with the <u>System76 Driver</u> application, the log files will be stored wit
 
 ### Sending the Logs to the Support Team
 
-Once the logs are created, you can either attach them directly to the support case, or you can email them to [careteam@system76.com](mailto:careteam@system76.com) and reference your case number.
+Once the logs are created, you can attach them directly to the support case.
+
+> **NOTE:** In semi-rare cases, error messages can grow logs files to a size too large for an email attachment. If that's the case, use a file share service like Google Drive or [Wormhole](https://wormhole.app/), and then include the share link in an email response to the support ticket.
```
[the remark this answers is not in the document]
Comment (storyaddict) [reply content/log-files.md:82]: "some cases" it is.

I like "file sharing service" better. Good note.

File: content/log-files.md [hunk 4]
```diff
@@ -19,26 +19,38 @@ tableOfContents: true
...
+
+![Settings-Logs](/images/system76-driver/logs-in-settings.png)
+
 ## Generating Log Files Using the System76 Driver Application
 
-The System76 Driver can be opened by clicking the <u>Activities</u> button in the top left (or pressing the Ubuntu or Pop key), then searching for 'System76.' Among the applications listed should be the <u>System76 Driver</u> application. Go ahead and click the icon to launch the application, and enter your password when prompted to do so.
+The System76 Driver can be opened by clicking the `Activities` or `Applications` button in the top left (or pressing the <kbd>SUPER</kbd>, then searching for "System76." Among the applications listed should be the <u>System76 Driver</u> application. Click the icon to launch the application, and enter your password when prompted to do so.
```
[the remark this answers is not in the document]
Comment (storyaddict) [reply content/log-files.md:39]: Whoops. Yes, I'll fix that. Thanks.

File: content/log-files.md [hunk 5]
```diff
@@ -19,26 +19,38 @@ tableOfContents: true
...
 
-Among the information collected by the log files includes:
+The information collected by the log files includes but is not limited to:
 
 * The installed operating system and active kernel version
 * A listing of the filesystems connected to your system, and the space used
 * Information about your firmware
 * A list of disks connected to your system, as well as connected USB devices
-* Sensors information showing temperatures and fanspeeds
+* Sensors information showing temperatures and fan-speeds
```
[the remark this answers is not in the document]
Comment (storyaddict) [reply content/log-files.md:28]: Probably could be either. I'll remove the hyphen.

File: content/log-files.md [hunk 6]
```diff
@@ -19,26 +19,38 @@ tableOfContents: true
 
 Log files help our support team learn more about your system, and can be a great benefit during troubleshooting.
 
-Among the information collected by the log files includes:
+The information collected by the log files includes but is not limited to:
```
[the remark this answers is not in the document]
Comment (storyaddict) [reply content/log-files.md:22]: I think we should just give a complete list. I agree.

**Review by ahoneybun — APPROVED** [review]
The way that the "Create Log Files" button is cut off is a bit odd:

ksnip_20220413-115622

It doesn't break anything on it's own and other then that it looks good.

**Review by Ampersandstorm — APPROVED** [review]

**Pull request merged**